### DISCUSSION TITLE
v0.15.1

### DISCUSSION THREAD

**Discussion by StevenACoffman** [99designs/gqlgen#1769] [Announcements]
## What's Changed
* Fixed broken link by @mkusaka in https://github.com/99designs/gqlgen/pull/1768
* Fix #1765: Sometimes module info not exists or not loaded. by @ipfans in https://github.com/99designs/gqlgen/pull/1767

## New Contributors
* @mkusaka made their first contribution in https://github.com/99designs/gqlgen/pull/1768

**Full Changelog**: https://github.com/99designs/gqlgen/compare/v0.15.0...v0.15.1

<hr /><em>This discussion was created from the release <a href='https://github.com/99designs/gqlgen/releases/tag/v0.15.1'>v0.15.1</a>.</em>

**Comment by ItalyPaleAle** [comment]
Thanks for all the work you've been doing on gqlgen, and I'm using this to build my first project that uses GraphQL.

I would like to ask however if you could please consider making a changelog that is easier to digest, especially with larger releases such as 0.15.0. As someone that is not deeply familiar with the inner-workings of the projects, it would be nice to have the highlights of the release (what's breaking at the top, what's new, most relevant fixes). An example can be found in the ["Keep a changelog"](https://keepachangelog.com/en/1.0.0/) specification that you're referencing

**Reply by StevenACoffman** [reply]
I think everyone agrees that would be best, but it's just a question of who would volunteer their time to do it. I added this release as a discussion in the hopes that someone else would volunteer to summarize this release in exactly the way you are wishing for. Please take a stab at it!

In the mean time, I deliberately used git-chglog to automate generating the changelog, rather than spending the time to manually curate a summary, simply because it was more expedient for me.

I would also say the overwhelming nature of the changes between v0.14.x and v0.15.0 is owed more to the long gap between those releases. I would hope that in the future we will make more regular releases with more incremental changes.

**Reply by ipfans** [reply]
@StevenACoffman Hi, if there can self-recommend, I think I can be the volunteer to help manage this repo. And we are migrating into gqlgen federation, if there are some more experiences from khan academy would be more helpful.

**Reply by ItalyPaleAle** [reply]
> I think everyone agrees that would be best, but it's just a question of who would volunteer their time to do it.

I 100% understand and sympathize with the fact that as a maintainer you already have enough on your plate. Thanks for taking the time to respond.
